logic/src/main/java/org/itsallcode/whiterabbit/logic/service/AppService.java

```diff
@@ -44,6 +45,21 @@ public void update() {
 		} else {
 			LOG.info("Today {} is a {}, no update required", today, day.getType());
 		}
+		return day;
+	}
+
+	public void incrementInterruption(Duration additionalInterruption) {
+		if (additionalInterruption.isZero()) {
+			LOG.info("Interruption is zero: ignore.");
+			return;
+		}
+		final LocalDate today = clock.getCurrentDate();
+		final MonthIndex month = storage.loadMonth(today);
+		final DayRecord day = month.getDay(today);
+		final Duration totalInterruption = day.getInterruption().plus(additionalInterruption);
```

Comment:
This would better be called `updatedInterruption` because you can't tell at that point whether or not it is going to be the total.